test/kuttl/tests/glance_disk_formats/02-cleanup.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      oc kustomize ../../../../config/samples/layout/single | oc delete -n $NAMESPACE -f -
```

Comment:
L5 here can be changed to be:

```
oc kustomize ../../../../config/samples/layout/disk_format | oc delete -n $NAMESPACE -f -
```
after you move everything under `layout` directory.